lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -187,6 +188,36 @@ def add_processing_data(post, options)
         post[:processing] = options[:processing]
       end
 
+      def add_sender_data(post, options)
+        return unless options[:sender].is_a?(Hash)
+
+        sender = options[:sender]
+
+        post[:sender] = {}
+        post[:sender][:type] = sender[:type] if sender[:type]
+        post[:sender][:first_name] = sender[:first_name] if sender[:first_name]
+        post[:sender][:last_name] = sender[:last_name] if sender[:last_name]
+        post[:sender][:dob] = sender[:dob] if sender[:dob]
+        post[:sender][:reference] = sender[:reference] if sender[:reference]
+
+        if address = sender[:address]
+          post[:sender][:address] = {}
+          post[:sender][:address][:address_line1] = address[:address1] if address[:address1]
```

Comment:
re: the discrepancy between `address1` and `address_line_1` (and 2) between the original sender and recipient methods, I kinda feel like we should be able to accept the fields as they are documented in the Checkout docs. 

But since we already may have `address1` being sent based on our docs, could we make it so either convention is supported?

I'm envisioning something like 
```
address1 = address[:address1] || address[:address_line_1]
```
inserted just after the guard clause at 203, and then on lines 205-6 you can reference use those variables, so either way we are passing the details through

Reply:
good idea @jcreiff ! I also opened the recipient field to accept both field names as well for the addresses